docs/guides/features/icons.md

```diff
@@ -44,8 +61,18 @@ If you need to source your icons from some other location, you can edit this par
 
 :::info Note on filetypes
 
-- icon.icns = macOS
-- icon.ico = MS Windows
-- \*.png = Linux
+- `icon.icns` = macOS
+- `icon.ico` = Windows
+- `*.png` = Linux
+- `Square*Logo.png` & `StoreLogo.png` = Currently unused but intended for AppX/MS Store targets.
+
+Note that especially the png icons may be used on other platforms. Therefore we recommend including all icons even if you intend to only build for a subset of platforms.
 
 :::
+
+If you plan on building these icons yourself, for example because you don't want to depend on the CLI's itnernal image resizing, especially if you want to have simpler design for small sizes, the required layer sizes and names for the [ICNS] file are described [here] and the [ICO] file must include layers for 16, 24, 32, 48, 64 and 256px.  
```

Comment:
```suggestion
If you prefer to build these icons yourself (if you want to have simpler design for small sizes or because you don't want to depend on the CLI's internal image resizing), the required layer sizes and names for the [`icns`] file are described [in the Tauri repo] and the [`ico`] file must include layers for 16, 24, 32, 48, 64 and 256 pixels.
```